lib/java/ydoc-server/src/test/java/org/enso/ydoc/polyfill/web/TimersTest.java

```diff
@@ -119,7 +119,7 @@ public void clearInterval() throws Exception {
         var p = function (x, y) {
             globalThis.result += 10*x + y;
         };
-        var intervalId = setInterval(p, 10, 4, 2);
+        var intervalId = setInterval(p, 99999, 4, 2);
```

Comment:
I'm surprised it worked at all. 10ms is a tight gap

Reply:
In most JS environments, even 0 here would be guaranteed to not actually run the callback if you clear it synchronously within the same task. Also 10ms sounds like a lot for most things, that's more than 1 frame budget for 120hz rendering.

Reply:
> even 0 here would be guaranteed to not actually run the callback if you clear it synchronously within the same task

In such case we probably need to fix the task scheduler and not to change the test...

Reply:
The merger was faster than me...

@4e6 please [fix the problem again](https://github.com/enso-org/enso/pull/10118#issuecomment-2136490798) and `setInterval` to `1` or `0` to verify the fix is correct.